Makefile

```diff
@@ -1,22 +1,25 @@
+
 deps: setup-env deps-venv2 deps-venv3
 
 setup-env: venv2 venv3 
 
 venv2:
 	mkdir -p venv2
-	cd venv2
 	pyenv install -s 2.7.16
-	pyenv local 2.7.16
-	cd ..
-	virtualenv venv2
+ifneq "$(shell PYENV_VERSION=2.7.16 python -V 2>&1)" "Python 2.7.16"
+	echo "pyenv not configured correctly, check your configuration"
+	exit 1
+endif
+	PYENV_VERSION=2.7.16 virtualenv venv2
 
 venv3:
 	mkdir -p venv3
-	cd venv3
 	pyenv install -s 3.7.4
-	pyenv local 3.7.4
-	cd ..
-	python -m venv venv3 
+ifneq "$(shell PYENV_VERSION=3.7.4 python -V 2>&1)" "Python 3.7.4"
+	echo "pyenv not configured correctly, check your configuration"
```

Comment:
I'm hitting this statement on both macOS 10.14 and macOS 10.15.  On both I had previously installed pyenv with `brew install pyenv`.

After digging around a bit in how to use pyenv, I was able to get it to work by changing the previous line to `ifneq "$(shell PYENV_VERSION=3.7.4 pyenv exec python -V 2>&1)" "Python 3.7.4"`, as well as changing lines 13 and 22 to include the `pyenv exec` statement as well.